stacker_blueprints/aws_lambda.py

```diff
@@ -219,3 +223,44 @@ def create_template(self):
         self.create_role()
         self.create_function()
         self.create_policy()
+
+
+class FunctionScheduler(Blueprint):
+
+    VARIABLES = {
+        "CloudwatchEventsRule": {
+            "type": TroposphereType(events.Rule),
+            "description": "The troposphere.events.Rule object params.",
+        },
+    }
+
+    def create_scheduler(self):
+        variables = self.get_variables()
+        troposhere_events_rule = variables["CloudwatchEventsRule"]
+        aws_lambda_arns = []
+
+        # append the subject Lambda Function ARN to event rule targets.
+        if "Targets" in troposhere_events_rule.properties:
+            for target in troposhere_events_rule.properties["Targets"]:
+                if target.properties["Arn"].startswith("arn:aws:lambda"):
```

Comment:
Probably worth adding the last colon, just in case there is another arn type someday that is "lambdasomething".

`"arn:aws:lambda:"`